### PR TITLE
Remove create alias

### PR DESCRIPTION
**Pull request by taquitos**
Resolves [CSDK-152]

Deprecated in RCv3, removed in RCv4 🎉

[CSDK-152]: https://revenuecats.atlassian.net/browse/CSDK-152?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Sources/Error Handling/ErrorCode.swift [hunk 1]
```diff
@@ -59,7 +58,19 @@ import Foundation
...
     @objc(RCOfflineConnectionError) case offlineConnectionError = 35
 
+    /**
+     * When an ErrorCode has been deprecated and then removed, add it to the reserved list so that we do not
+     * accidentally reuse it. For example:
+     * `@objc(RCMissingAppUserIDForAliasCreationError) case missingAppUserIDForAliasCreationError = 27` was removed,
+     * so we add its rawValue of `27` to the `reservedRawValues` array. That way our unit tests will catch if we
+     * accidentally add `27` back into the enumeration.
+     */
+    static var reservedRawValues: [Int] {
```
Comment (taquitos) [inline Sources/Error Handling/ErrorCode.swift:68]: Making sure we don't reuse any values in the future 😄

Comment (NachoSoto) [inline Sources/Error Handling/ErrorCode.swift:68]: Nit: maybe move this to an extension?

Comment (NachoSoto) [reply Sources/Error Handling/ErrorCode.swift:68]: I love this idea though.

Comment (NachoSoto) [reply Sources/Error Handling/ErrorCode.swift:68]: Should be a set though, and maybe use `RawValue` instead?
```suggestion
    static var reservedRawValues: Set<RawValue> {
```

File: Tests/UnitTests/Purchasing/ErrorCodeTests.swift [hunk 2]
```diff
@@ -173,11 +169,15 @@ class ErrorCodeTests: TestCase {
...
-        expect(ErrorCode.allCases).to(haveCount(36))
+        expect(ErrorCode.allCases).to(haveCount(35))
     }
 
     func ensureEnumCaseMatchesExpectedRawValue(errorCode: ErrorCode, expectedRawValue: Int) {
         expect(errorCode.rawValue).to(equal(expectedRawValue))
     }
 
+    func testRemovedErrorCodesAreNotReAdded() {
+        expect(ErrorCode.reservedRawValues).toNot(contain(ErrorCode.allCases.map { $0.rawValue }))
```
Comment (NachoSoto) [inline Tests/UnitTests/Purchasing/ErrorCodeTests.swift:180]: Awesome.

**Review by NachoSoto — APPROVED** [review]
Awesome!
I think you can remove the snapshot files for those requests too :)

File: Sources/Error Handling/ErrorCode.swift (hunk 1 above)
Comment (taquitos) [reply Sources/Error Handling/ErrorCode.swift:68]: ✅

**Review by NachoSoto — APPROVED** [review]

**Pull request merged**